packages/core-js/internals/fix-regexp-well-known-symbol-logic.js

```diff
@@ -16,13 +16,17 @@ module.exports = function (KEY, length, exec) {
     return ''[KEY](O) != 7;
   })) {
     redefine(String.prototype, KEY, stringMethod);
-    hide(RegExp.prototype, SYMBOL, length == 2
+    redefine(RegExp.prototype, SYMBOL, length == 2
       // 21.2.5.8 RegExp.prototype[@@replace](string, replaceValue)
       // 21.2.5.11 RegExp.prototype[@@split](string, limit)
       ? function (string, arg) { return regexMethod.call(string, this, arg); }
       // 21.2.5.6 RegExp.prototype[@@match](string)
       // 21.2.5.9 RegExp.prototype[@@search](string)
       : function (string) { return regexMethod.call(string, this); }
     );
+    // TODO: This line makes the tests fail:
+    //       ReferenceError: Can't find variable: Reflect
+    // hide(RegExp.prototype[SYMBOL], 'name', '[Symbol.' + KEY + ']');
```

Comment:
In many engines, functions `.name` property non-writable non-configurable, so, I think, it would be better just remove it.